### ISSUE TITLE
[wait on backend] Add CxCmsComponentContainer cms component (comes from personalization)

### ISSUE THREAD

**Issue by Xymmer** [SAP/cloud-commerce-spartacus-storefront#961]
We have a special recipe that comments out smartedit and personalization extensions.

Wei says that personalization adds a cms component (CxCmsComponentContainer) we don't handle and that's why the some banners don't appear.

TBD: what is missing, can we add it? then we don't have to publish special recipes AND we are charged with getting SmartEdit working anyway.

**Comment by Xymmer** [comment]
what we do here is changing, so taking this back for now

**Comment by Xymmer** [comment]
new ticket is #1694
